modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/js/ManagementToolbarPropsTransformer.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+export default function propsTransformer({portletNamespace, ...otherProps}) {
+	return {
+		...otherProps,
+		onActionButtonClick(event, {item}) {
+			const action = item.data?.action;
+
+			if (action === 'deleteEntries') {
+				if (
+					confirm(
+						Liferay.Language.get(
+							'are-you-sure-you-want-to-delete-the-selected-entries'
+						)
+					)
+				) {
+					const form = document.getElementById(
+						`${portletNamespace}fm`
+					);
+
+					if (!form) {
```

Comment:
See previous comment in https://github.com/liferay-lima/liferay-portal/pull/1507#discussion_r577635010



> See https://github.com/liferay-frontend/liferay-portal/pull/759#discussion_r574501571
> 
> If it's just one line in if block, _submitForm_ in this case, it's hard to justify an early return.

Reply:
I'll do the change, but I think that we have defined so many rules that it is becoming more difficult to remember when to apply them